stubs/fpdf2/fpdf/text_region.pyi

```diff
@@ -22,14 +23,24 @@ class LineWrapper(NamedTuple):
     first_line: bool = False
     last_line: bool = False
 
+class Bullet:
+    fragments: Iterable[Fragment]
```

Comment:
Iterable seems like a dangerous type for an attribute; if it's a non-Sequence iterable `get_fragments_width()` may not return the same thing each time. Probably better to use Sequence.